src/components/AddTodo/AddTodo.jsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from 'react';
+import { addTask, deleteTasks } from 'components/Service/service';
+import './style.scss';
+
+const AddTodo = ({allTasks, setAllTasks}) => {
+  const [userInput, setUserInput] = useState('');
+
+  const onChange = (e) => {
```

Comment:
Is it necessary to put it in a separate function? Can this be done below here onChange={onChange}? That is
onChange={(e) => onChange(e.target.value} ??